src/EncompassApi.FuncApp/Functions/GetToken.cs

```diff
@@ -27,8 +29,15 @@ public TokenFunction (IEncompassApiClient client, ILogger<TokenFunction> log)
             _log.LogInformation("Token Service Validation start");
 
             //this call will fail but should call correct handlers to validate token
-            var result = _client.Webhook.GetSubscriptionsAsync();
 
+            List<Webhook.WebhookSubscription> result = new List<Webhook.WebhookSubscription>();
+            try
+            {
+                result.AddRange(await _client.Webhook.GetSubscriptionsAsync());
+            }catch(Exception ex)
+            {
+
```

Comment:
empty try/catch should be avoided

Reply:
This is just a function for testing. It is not automated. It won't hurt anything. It is meant to be watched by the developer.

Reply:
Merging as it is a test and eating the exception ensures function exit